internal/controllers/clusterextension_controller.go

```diff
@@ -121,17 +121,14 @@ func (r *ClusterExtensionReconciler) reconcile(ctx context.Context, ext *ocv1alp
 	// Lookup the bundle that corresponds to the ClusterExtension's desired package.
 	bundle, err := r.resolve(ctx, ext)
 	if err != nil {
-		ext.Status.InstalledBundle = nil
```

Comment:
Re: keeping this fields in ClusterExtension, we also want to keep the ClusterExtension reconciler logic intact.

Reply:
Ok. Ack. Keeping it intact.